MsCorePkg/MacAddressEmulationDxe/MacAddressEmulationDxe.c

```diff
@@ -0,0 +1,239 @@
+/** @file
+
+  DXE Driver for handling MAC Address Emulation.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include "MacAddressEmulationDxe.h"
+
+#include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/DebugLib.h>
+#include <Library/UefiLib.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/NetLib.h>
+
+#include "Library/MacAddressEmulationPlatformLib.h"
```

Comment:
need to use the include path version <> instead of "".  That is not a local file.